src/css/custom.scss

```
@@ -264,3 +264,14 @@
 .navbar__logo {
   width: 2rem;
 }
+
+// blog sidebar custom styles
+[aria-label*='Blog recent posts navigation'] {
+  padding: 0.5rem;
+  & [class*='yearGroupHeading_'] {
+    margin: 0.75rem 0 0.5rem;
+    color: var(--subtle);
+    border-bottom: 0.01rem solid var(--ifm-table-border-color);
+    padding-bottom: 0.2rem;
+  }
+}
```

```suggestion
.plugin-blog [class*='sidebar_'] {
  padding: 0.5rem;
  & [class*='yearGroupHeading_'] {
    margin: 0.75rem 0 0.5rem;
    color: var(--subtle);
    border-bottom: 0.01rem solid var(--ifm-table-border-color);
    padding-bottom: 0.2rem;
  }
}
```

I don't love using the `aria-label` as a selector, that's going to be pretty fragile. Looks like the blog page has a `plugin-blog` class added to the `html` element, let's use that instead along with the `sidebar_` class.